src/pages/project/project.jsx

```diff
@@ -82,7 +82,11 @@ var Project = React.createClass({
     // FIXME: TODO: This should be done by preventDefaulting the touch event, not via CSS.
     document.body.style.overflowY = 'hidden';
     var mode = this.state.params.mode;
-    var isPlayOnly = (mode === 'play' || mode === 'link');
+
+    var isPlayOnly = mode === 'play';
```

Comment:
small nit: this is confusing to read because of how JS works (allowing things like `var a = b = c = value`, so a `var isPlayOnly = (mode === 'play');` is a lot easier to immediately read.